routers/web/repo/pull.go

```diff
@@ -113,9 +113,9 @@ func getForkRepository(ctx *context.Context) *models.Repository {
 
 	ctx.Data["ForkRepo"] = forkRepo
 
-	ownedOrgs, err := models.GetOwnedOrgsByUserID(ctx.User.ID)
+	ownedOrgs, err := models.GetOrgsCanCreateRepoByUserID(ctx.User.ID)
```

Comment:
There is an performance regression for all pages with fork button. This should be fixed in v1.16.